demos/basics/loading-pandas.ipynb

```diff
@@ -0,0 +1,3961 @@
+{
```

Comment:
<p>Could remove "To start with" at the beginning of the sentence, since it's immediately followed by "we'll start"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1074/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>